concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md

```diff
@@ -2,7 +2,7 @@
 
 ### Obtain Your License Key
 
-Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCenter/DownloadManager/). Find your DevExtreme subscription license in the product list. Expand the product entry to access available downloads and additional information. Follow on-screen instructions to copy your key.
+Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCenter/DownloadManager/). Find your **DevExtreme Subscription** license in the product list (even if you have a Universal, DXperience, or ASP.NET/Blazor subscription, the key is under the **DevExtreme Subscription** entry). Expand the product entry to access available downloads and additional information. Follow on-screen instructions to copy your key.
 
```

Comment:
Can you update the following line below?

Line 13:  
`Below there are instructions on how to set up a license key based on whether your project is *modular* or *non-modular*.`

Please update to:

`See the instructions below for information on how to set up a license key based on whether your project is *modular* or *non-modular*.`